qutebrowser/browser/commands.py

```diff
@@ -25,7 +25,9 @@
 import subprocess
 import posixpath
 import functools
+from xml.etree import ElementTree
```

Comment:
I'd prefer just `import xml.etree` here and then using `xml.etree.ElementTree` etc. so it's clear which module this is coming from.